content/applications/inventory_and_mrp/inventory/management/lots_serial_numbers/lots.rst

```diff
@@ -1,137 +1,221 @@
+    :abbr:`FEFO (First Expiry, First Out)`). It will also depend on the quantity ordered, and if
+    there is enough quantity in one lot to fulfill the order.
+
+Then, click :guilabel:`Add a line`, select an additional (different) :guilabel:`Lot/Serial Number`,
+apply the rest of the :guilabel:`Done` quantities, and click :guilabel:`Confirm` to close the
+popup. Click :guilabel:`Validate` to deliver the products.
+
+.. image:: lots/lots-detailed-operations-popup.png
+   :align: center
+   :alt: Detailed operations popup for source lot number on sales order.
+
+A :guilabel:`Traceability` smart button will appear upon validating the delivery order. Click the
+:guilabel:`Traceability` smart button to see the updated :guilabel:`Traceability Report`, which
+includes: a :guilabel:`Reference` document; the :guilabel:`Product` being traced; the
+:guilabel:`Lot/Serial #` assigned; and the :guilabel:`Reference` receipt from the previous purchase
+order (if the product quantities shared the same lot number).
+
+Manage lots for different operations types
+==========================================
+
+By default in Odoo, the creation of new lots is only allowed upon *receiving* products from a
+purchase order. Also by default, using *existing* lot numbers isn't allowed upon receipt. For sales
+orders, it is the opposite: new lot numbers cannot be created on the delivery order, but existing
+lot numbers can be used.
+
+To change the ability to use new or existing lot numbers on any operation type, go to
+:menuselection:`Inventory --> Configuration --> Operations Types`, and select the desired
+:guilabel:`Operation Type`. For :guilabel:`Receipts`, the
+:guilabel:`Use Existing Lots/Serial Numbers` option can be enabled by clicking :guilabel:`Edit` and
+then clicking the :guilabel:`checkbox`. For :guilabel:`Delivery Orders`, the
+:guilabel:`Create New Lots/Serial Numbers` option can be enabled by clicking :guilabel:`Edit` and
+clicking the checkbox. For any changes made, be sure to :guilabel:`Save`.
+
+.. image:: lots/lots-operations-type-form.png
+   :align: center
+   :alt: Enabled traceability setting on operations type form.
+
+.. tip::
+    For inter-warehouse trasnfers involving products tracked by lots, it can be useful to enable
+    the :guilabel:`Use Existing Lots/Serial Numbers` option for :guilabel:`Receipts`.
 
 Lots traceability
 =================
 
-The lot number given to the products allows you to keep track of where
-they were received, put in stock, to whom they were sold and where they
-were shipped to.
-
-To track an item, open the *Inventory* module and, in :menuselection:`Master Data -->
-Lots/Serial Numbers`, click on the lot number corresponding to your
-search.
-
-.. image:: lots/lots_09.png
-    :align: center
-
-When you open the *Traceability* information, you see in which
-documents the lot number has been used.
+The benefit of using lots (or serial numbers) to track products is that it offers end-to-end
```

Comment:
Instead of describing the benefits of lots overall, this can be one sentence explaining the traceability report. "The traceability report shows x, y, and z". Otherwise it sounds a bit too much like an introduction to a new document.